src/Behat/YiiExtension/ServiceContainer/YiiExtension.php

```diff
@@ -22,21 +23,29 @@
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  */
-class Extension extends BaseExtension
+class YiiExtension implements Extension
 {
+    public function getConfigKey()
+    {
+        return 'yii_extension';
+    }
+
     /**
      * Loads a specific configuration.
      *
      * @param array            $config    Extension configuration hash (from behat.yml)
      * @param ContainerBuilder $container ContainerBuilder instance
      */
-    public function load(array $config, ContainerBuilder $container)
+    public function load(ContainerBuilder $container, array $config)
     {
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/services'));
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../services'));
         $loader->load('yii.xml');
-        $basePath = $container->getParameter('behat.paths.base');
 
-        $extensions = $container->getParameter('behat.extension.classes');
+        // echo json_encode($container->getParameterBag()->all()); exit;
+
+        $basePath = $container->getParameter('paths.base');
+
+        $extensions = $container->getParameter('extensions');
```

Comment:
this is used by the WUnitDriver registration, but the way it is done currently is wrong for MinkExtension 2. Registering a session is the wrong way. It should register a driver factory instead (look at the Symfony2Extension for an example)